src/components/SchedulePage/StickyTimeSlot.js

```diff
@@ -27,6 +27,8 @@ const useStyles = makeStyles(theme => {
 			}
 		},
 		eventItem: {
+			paddingTop: theme.spacing(3),
+			paddingBottom: theme.spacing(8),
```

Comment:
Sorry, this is deep into bikeshedding territory, but 

```suggestion
			marginTop: theme.spacing(2),
			marginBottom: theme.spacing(6),
```

looks a bit better to me. padding → margin, since margins collapse (the margin-bottom of the previous item and the margin-top of the next item overlap). The spacing numbers are just bikeshedding.